### ISSUE TITLE
Crash somewhere

### ISSUE THREAD

**Issue by miszmaniac** [journeyapps/zxing-android-embedded#241]
#### Description of the problem:
I found that on Crashlytics, but there is no more debug info there:
```
Fatal Exception: java.lang.NullPointerException
       at com.journeyapps.barcodescanner.SourceData.(SourceData.java)
       at com.journeyapps.barcodescanner.camera.CameraManager$CameraPreviewCallback.onPreviewFrame(CameraManager.java:101)
       at android.hardware.Camera$EventHandler.handleMessage(Camera.java:986)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:146)
       at android.os.HandlerThread.run(HandlerThread.java:61)

```
**Which library version are you using?** 3.4.0.

**Which phone/tablet are you using, and which Android version does it run? (e.g. Samsung Galaxy S5,
  Android 5.0)** Samsung GT-I9305 Android 4.4.4

Don't have more info about issue, but maybe it will help you somehow:)

**Comment by rkistner** [comment]
Thanks, will investigate.

**Comment by rkistner** [comment]
Fixed in 9ec92a510102a2024acc13a596a76f59ccf6616e.
